tests/converters/table-post-fixer.js

```diff
@@ -181,6 +181,19 @@ describe( 'Table post-fixer', () => {
 		} );
 	} );
 
+	describe( 'on removing tableCell contents', () => {
+		it( 'should fix empty table cells', () => {
```

Comment:
It's hard to understand from this test that it checks that a paragraph should be added. I checked that it fails without the patch and the error is quite obvious then, but the code and description of this test are cryptic. 

There are many opinions regarding writing so many tools for creating tests. But, leaving this aside, when we write them we should make sure that tests remain understandable.